lazyfs/scripts/mount-lazyfs.sh

```diff
@@ -72,8 +72,8 @@ echo -e "Running LazyFS (stop with ctrl+c or umount-lazyfs.sh)...\n"
 if [ -z "$CMD_FG" ]; then
    echo -e "(running in no foreground mode)\n"
    echo -e "Note: run in foreground to see the <stdio> logs.\n"
-   ./build/lazyfs $MOUNT_DIR --config-path $CMD_CONFIG -o allow_other -o modules=subdir -o subdir=$ROOT_DIR &
+   ./build/lazyfs $MOUNT_DIR --config-path $CMD_CONFIG -o allow_other -o modules=subdir -o subdir=$ROOT_DIR -s &
```

Comment:
Why are you forcing single-threaded mode in the mount script? Do you expect problems when multithreading is enabled?

Reply:
I forgot to remove that! I wanted to check the new faults with single-threaded mode